src/lumigo_tracer/libs/awslambdaric/release_2_0_0/runtime_client.cpp

```diff
@@ -0,0 +1,148 @@
+/* Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved. */
```

Comment:
I don't think that we should have code that we don't use. I would trim all these new files only to the bare necessary

Reply:
Removed